library/webpack-loaders/react-universal-server-loader.js

```diff
@@ -25,9 +25,12 @@ function createServerCode({ importPath, scriptSrc, id }) {
        |\` }} />`.split(/^[ \t]*\|/m).join('')
 
   return `|import Component from './${importPath}'
+          |import hoistNonReactStatic from 'hoist-non-react-statics'
           |import { renderToString } from 'react-dom/server'
           |
-          |export default function WrapWithScript(props) {
+          |export default hoistNonReactStatic(WrapWithScript, Component)
```

Comment:
Could you add comments like this: `hoistNonReactStatic(/* target */ WrapWithScript, /* source */ Component)`?

It's confusing, normally it's source first, target second. Ehhh, scrap that, for object assign it's similar. Maybe just rename `hoistNonReactStatic` to `assignStatics`.